platform/android/os_android.cpp

```diff
@@ -393,7 +420,11 @@ Error OS_Android::shell_open(String p_uri) {
 }
 
 String OS_Android::get_resource_dir() const {
+#ifdef TOOLS_ENABLED
+	return OS::get_resource_dir();
```

Comment:
It's not defined currently but it might be safer to use `OS_Unix::get_resource_dir()`?